spec/models/concerns/clearable_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Clearable do
+  let(:mocked_model) do
+    Struct.new('Commodity') do
+      include Clearable
+
+      def self.association_reflections
+        @association_reflections ||=
+          begin
+            {
+              foo: { cache: { rows: ['some stuff'] } },
+              bar: { cache: {} },
+              baz: { flibble: { some_option: true } },
+
+            }
+          end
```

Comment:
Do we need `begin..end` here?

Reply:
I could move to instantiating the variable outside of the getter and remove the begin/end, here.